testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -262,15 +267,15 @@ def test_covariance_matrix_with_reference(self, ens1):
                                                          select="name CA and resnum 1:3",
                                                          estimator=encore.covariance.shrinkage_covariance_estimator,
                                                          reference=ens1)
-        assert_almost_equal(covariance, reference_cov, decimal=4,
-                            err_msg="Covariance matrix from covariance estimation not as expected")
+        err_msg="Covariance matrix from covariance estimation not as expected"
+        assert_almost_equal(covariance, reference_cov, 4, err_msg)
 
     def test_hes_to_self(self, ens1):
         results, details = encore.hes([ens1, ens1])
         result_value = results[0, 1]
-        expected_value = 0.
+        expected_value = 9.
```

Comment:
Why was this change made? It's making the tests fail.
```suggestion
        expected_value = 0.
```

Reply:
It was an error made during debugging. I have corrected in latest commit